pkgs/applications/science/math/engauge-digitizer/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, stdenv, fetchFromGitHub, fftw, libjpeg, log4cpp, openjpeg, libpng12, libsForQt5 } :
+
+stdenv.mkDerivation rec {
+  pname = "engauge-digitizer";
+  version = "12.1";
+
+  src = fetchFromGitHub {
+    owner = "markummitchell";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "12gflxqaa4w6vifwpaqwl0l3f9qq5pbrh45s1rpc584nmr0897a7";
+  };
+
+  nativeBuildInputs = with libsForQt5; [ qmake wrapQtAppsHook ];
+
+  buildInputs = with libsForQt5; [
```

```suggestion
  nativeBuildInputs = [ qmake wrapQtAppsHook ];

  buildInputs = [
```
Please list the required package in inputs.